spec/puppet-server-lib/puppet/jvm/puppet_config_spec.rb

```diff
@@ -21,13 +21,6 @@
       end
     end
 
-    describe '(SERVER-410) Puppet[:always_cache_features]' do
-      subject { Puppet[:always_cache_features] }
-      it 'is true for increased performance in puppet-server' do
-        expect(subject).to eq(true)
-      end
-    end
-
```

Comment:
We just did this in person, but I'll jot it down and update the comment.
Having `always_retry_plugins` be false will imply the behavior that `always_cache_features = true` would cause, but doesn't actually change the value of the `always_cache_features` setting.